database/migrations/2021_04_06_113426_printcat_ora_setup.php

```diff
@@ -13,7 +13,7 @@ class PrintcatOraSetup extends Migration {
      */
     public function up() {
 
-        if (!Schema::hasTable('devices_faults_printers_ora_opinions')) {
+        if (!Schema::hasTable('devices_faults_tablets_ora_opinions')) {
```

Comment:
As this is the migration for PrintCat, the table it is checking for should be `devices_faults_printers_ora_opinions`.